network.koesling.shm-modbus.yml

```diff
@@ -0,0 +1,128 @@
+id: network.koesling.shm-modbus
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: shm-modbus
+finish-args:
+  - --device=shm
+  - --share=network
+modules:
+  - name: launch_scripts_and_metadata
+    buildsystem: simple
+    build-commands:
+      - install -D -m 755 shm-modbus ${FLATPAK_DEST}/bin/shm-modbus
+      - install -D -m 644 guided-commands.sh ${FLATPAK_DEST}/bin/guided-commands.sh
+      - install -D -m 644 manual-commands.sh ${FLATPAK_DEST}/bin/manual-commands.sh
+
+      - install -D -m 644 network.koesling.shm-modbus.metainfo.xml "${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.metainfo.xml"
+      - install -D -m 644 -t "${FLATPAK_DEST}/share/icons/hicolor/scalable/apps" ${FLATPAK_ID}.svg
+      - install -D -m 644 ${FLATPAK_ID}_256.png "${FLATPAK_DEST}/share/icons/hicolor/256x256/apps/${FLATPAK_ID}.png"
+      - install -D -m 644 -t ${FLATPAK_DEST}/share/applications/ ${FLATPAK_ID}.desktop
+
+    sources:
+      - type: git
+        tag: v1.0.0
+        commit: e52ff54f53a7d747eac57bdf627f315d5686e132 
+        url: https://github.com/NikolasK-source/SHM_Modbus.git
+
+  - name: Modbus_TCP_client_shm
+    buildsystem: simple
+    build-commands:
+      # build
+      - mkdir build
+      - cmake -B build . -DCMAKE_BUILD_TYPE=Release -DCLANG_FORMAT=OFF -DCOMPILER_WARNINGS=OFF
+      - cmake --build build
+
+      # install
+      - install -D -m 755 build/modbus-tcp-client-shm ${FLATPAK_DEST}/bin/modbus-tcp-client-shm
+
+    sources:
+      - type: git
+        tag: v1.1.1
+        commit: ef7aa067fce9af43cf9751440be852b8766e0a6d 
+        url: https://github.com/NikolasK-source/modbus_tcp_client_shm.git
+
+  - name: Modbus_RTU_client_shm
+    buildsystem: simple
+    build-commands:
+      # build
+      - mkdir build
+      - cmake -B build . -DCMAKE_BUILD_TYPE=Release -DCLANG_FORMAT=OFF -DCOMPILER_WARNINGS=OFF
+      - cmake --build build
+
+      # install
+      - install -D -m 755 build/modbus-rtu-client-shm ${FLATPAK_DEST}/bin/modbus-rtu-client-shm
+
+    sources:
+      - type: git
+        tag: v0.2.0
+        commit: 5aa84184b9dbbb4d644686126b6d03b5f70bbb0c 
+        url: https://github.com/NikolasK-source/modbus_rtu_client_shm.git
+
+  - name: write_shm
+    buildsystem: simple
+    build-commands:
+      # build
+      - mkdir build
+      - cmake -B build . -DCMAKE_BUILD_TYPE=Release -DCLANG_FORMAT=OFF -DCOMPILER_WARNINGS=OFF
+      - cmake --build build
+
+      # install
+      - install -D -m 755 build/write_shm ${FLATPAK_DEST}/bin/write-shm
+
+    sources:
+      - type: git
+        tag: v0.1.2
+        commit: 89a0647f349b1c87d30e7ea1cb51e87f649a2531 
+        url: https://github.com/NikolasK-source/write_shm.git
+
+  - name: SHM_random
+    buildsystem: simple
```

Comment:
and here too and every other places I won't write out but expect to be addressed too.